File: src/primus/pawel/Exercise_013.java
package primus.pawel;

import java.util.Arrays;

// Highest and Lowest
// 7kyu
// https://www.codewars.com/kata/554b4ac871d6813a03000035/train/java
//
class Exercise_013 {

    public static String highAndLow(String numbers) {

        int lowest = Integer.MAX_VALUE;
        int highest = Integer.MIN_VALUE;
        numbers += " ";
        StringBuilder sB = new StringBuilder("");

        for (int i = 0; i < numbers.length(); i++) {
            if (numbers.charAt(i) == ' ') {
                //System.out.println("");
                // System.out.println(sB.toString());
                if (Integer.parseInt(sB.toString()) < lowest) {
                    lowest = Integer.parseInt(sB.toString());
                }
                if (Integer.parseInt(sB.toString()) > highest) {
                    highest = Integer.parseInt(sB.toString());
                }
                sB.delete(0, sB.length());
            } else {
                //System.out.print(numbers.charAt(i));
                sB.append(numbers.charAt(i));
            }
        }
        //System.out.println(highest);
        //System.out.println(lowest);
        return highest + " " + lowest;
    }

    public static void main(String[] args) {
        System.out.println(highAndLow2("8 3 -5 42 -1 0 0 -9 4 7 4 -4"));
    }

    public static String highAndLow2(String numbers) {
        String[] dividedNumbers = numbers.split("[ ]");
        StringBuilder sbResult = new StringBuilder();
        sbResult.append(Arrays.stream(dividedNumbers).mapToInt(Integer::parseInt).max().orElse(-1))
        .append(" ");
        sbResult.append(Arrays.stream(dividedNumbers).mapToInt(Integer::parseInt).min().orElse(-1));
        return sbResult.toString();
    }
}

File: src/JS/Exercise_013.js

// Sum The Strings
// 8kyu
// https://www.codewars.com/kata/5966e33c4e686b508700002d/train/javascript
// 2022-03-10 T:22:19:31
function sumStr(a, b) {
    let numA;
    let numB;

    if(a.length !== 0){
        numA = parseInt(a)
    } else {
        numA = 0;
    }

    if(b.length !== 0){
        numB = parseInt(b)
    } else {
        numB = 0;
    }
    return numA + numB  + ""
}

console.log(sumStr("4", "5"));